php_phongo.c

```diff
@@ -45,13 +45,15 @@
 #include <Zend/zend_exceptions.h>
 #include <ext/spl/spl_iterators.h>
 #include <ext/spl/spl_exceptions.h>
-/* For formating timestamp in the log */
-#include <ext/date/php_date.h>
 /* Stream wrapper */
 #include <main/php_streams.h>
 #include <main/php_network.h>
 /* Debug log writing */
 #include <main/php_open_temporary_file.h>
+/* For formating timestamp in the log */
+#include <ext/date/php_date.h>
+/* String manipulation */
+#include <Zend/zend_string.h>
```

Comment:
I don't quite see the benefit of making it conditional?
Its probably even implicitly added already, we use str_efree() from that header file, and phongo_compat.h seems to include it.... So maybe just remove it if you want to reduce the includes.